model/src/wav_import_export.F90

```diff
@@ -96,6 +96,7 @@ subroutine advertise_fields(importState, ExportState, flds_scalar_name, rc)
     ! local variables
     integer          :: n, num
     character(len=2) :: fvalue
+    logical          :: aux_flds_to_cmeps = .true.
```

Comment:
There is no way to turn this off?
I'm not thrilled about hard coding a behavior-changing (and cost increasing) modification.

Reply:
@gold2718 - I agree. I can add a new namelist variable to ww3dev to turn this off. Let me implement that instead.